redash/query_runner/dremio.py

```diff
@@ -0,0 +1,119 @@
+from redash.query_runner import BaseQueryRunner, register
+from redash.query_runner import TYPE_STRING, TYPE_DATE, TYPE_DATETIME, TYPE_INTEGER, TYPE_FLOAT, TYPE_BOOLEAN
+from redash.utils import json_dumps, json_loads
+import pyodbc
+
+enabled = True
+
+TYPES_MAP = {
+    0: TYPE_INTEGER,
+    1: TYPE_FLOAT,
+    2: TYPE_STRING,
+    3: TYPE_DATE,
+    4: TYPE_DATETIME,
+    5: TYPE_STRING,
+    6: TYPE_DATETIME,
+    13: TYPE_BOOLEAN
+}
+
+
+class Dremio(BaseQueryRunner):
+    noop_query = "SELECT 1"
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "driver": {
+                    "type": "string",
+                    "default": "{Dremio ODBC Driver 64-bit}"
```

Comment:
Any reason someone might want to change this?

Reply:
Depend on the operating system, in osx is different from linux:
- Dremio ODBC Driver 64-bit
- Dremio ODBC Driver 32-bit
- Dremio ODBC Driver

Reply:
Isn't it better to specify driver name in env var?